dev/README.md

```diff
@@ -18,6 +18,21 @@ pip install -e .[dev] # includes what is needed for tests and code development
 pip install -e .[admin] # dev plus what is needed for creating documentation and releasing packages
 ```
 
+**Java**
+Spark requires Java. Most people already have Java installed on their personal computers, so this
```

Comment:
NIT: local instead of personal? You used local computer or laptop later.

Reply:
I didn't write this originally and didn't want to change the author's words.